aptos-move/framework/src/extended_checks.rs

```diff
@@ -498,6 +498,31 @@ impl<'a> ExtendedChecker<'a> {
                 self.env
                     .error(&fun.get_id_loc(), "view function must return values")
             }
+
+            fun.get_parameter_types()
```

Comment:
Should we add this check to `check_transaction_args` where all parameter types are checked? cc @runtian-zhou

Reply:
check_transaction_args checks for main function so it shouldn't matter here?